# nav_msgs/Path MissionPath
geographic_msgs/GeoPath MissionPath
int32 h_d
---
bool MissionReceived